### ISSUE TITLE
Can't walk past wing mirrors

### ISSUE THREAD

**Issue by nornagon** [CleverRaven/Cataclysm-DDA#74404]
### Describe the bug

Wing mirrors mounted on vehicles are impassable obstacles. This doesn't seem right.

### Attach save file

[TESTING 2-trimmed.tar.gz](https://github.com/user-attachments/files/15749801/TESTING.2-trimmed.tar.gz)

### Steps to reproduce

1. Spawn a Luxury RV
2. Try to walk onto a tile with a wing mirror

### Expected behavior

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/172800/4d8ad61d-b06f-4d94-9fa3-3962b2927fae)

idk i reckon i could walk past it

### Versions and configuration

- OS: MacOs
    - OS Version: Name: macOS; Version: 14.3; Build: 23D56; 
- Game Version: 631dda72fe-dirty [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth]
]

**Comment by nornagon** [comment]
Did a little digging and the issue seems to be that there's a fake `windshield` vpart being generated for the tile with the wing mirror. Not sure if that's intentional or not.

**Comment by PGR-14** [comment]
I'm pretty sure those wing mirrors are those big ones on the side of busses & semis.

**Comment by nornagon** [comment]
wing mirrors on busses and semis are high above the ground and shouldn't block movement either.

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/172800/7a43b440-5679-4f8c-9b52-dae8136369f7)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/172800/f8fd5149-d503-46b5-9a8d-08edb2e5c6a0)

**Comment by PGR-14** [comment]
Yeah, fair, they used to have a speed penalty, which doesn't make much sense. We should probably make it so you can move freely.

**Comment by silvadomi** [comment]
Has this been fixed? In 0.H I can walk through wing mirrors, at a move cost of 400.

- OS: MacOs
    - OS Version: Name: <unknown>; Version: <unknown>; Build: <unknown>; 
- Game Version: 0.H-201-gbe7a566ffc [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Portal Storms Ignore NPCs [personal_portal_storms],
    Slowdown Fungal Growth [no_fungal_growth]
]

**Comment by AideeAmei** [comment]
It is actually quite a nuisance. Parking a van by the radio tower and getting out to close the gate means either walking around the whole vehicle or climbing up a fence to get past the wing mirror. It's ridiculous. I get that the mirror can get destroyed when clipping a tree or a house, but walking past it should be somewhat possible, even if with a movement penalty.
